### PR TITLE
Added Large File Storage step to instructions

### PR DESCRIPTION
**Pull request by jzaki**
## Issue Addressed

Which issue # does this PR address?

(minor change to readme)

Please list or describe the changes introduced by this PR.

Added Large File Storage step so EF tests pass

Please provide any additional information. For example, future considerations
or information useful for reviewers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,15 +120,18 @@ A few basic steps are needed to get set up:
...
 	  - `clang`: required by RocksDB.
 	  - `protobuf`: required for protobuf serialization (gRPC).
-	  - `cmake`: required for building protobuf
+	  - `cmake`: required for building protobuf.
+	  - `git-lfs`: The Git extension for [Large File Support](https://git-lfs.github.com/) (required for EF tests submodule).
    6. Navigate to the working directory.
-   7. Run the test by using command `cargo test --all`. By running, it will pass all the required test cases.
+   7. If you haven't already, clone the repository with submodules: `git clone --recursive https://github.com/sigp/lighthouse`.
+		Alternatively, run `git submodule init` in a repository which was cloned without submodules.
+   8. Run the test by using command `cargo test --all`. By running, it will pass all the required test cases.
```
Comment (paulhauner) [inline README.md]: I think we should make this command `cargo test --all --release`.

The tests run much, much faster when compiled to release :)

Comment (jzaki) [reply README.md]: Sounds good to have a faster sanity check when compiling for the first time.

Comment (paulhauner) [reply README.md]: Just checking, do you intend to add the `--release` flag?

Comment (jzaki) [reply README.md]: Done

**Pull request merged**